specifyweb/frontend/js_src/lib/components/QueryBuilder/FieldFilter.tsx

```diff
@@ -560,6 +564,24 @@ export const queryFieldFilters: RR<
     types: ['checkbox'],
     hasParser: true,
   },
+  name: {
+    id: 15,
+    label: resourcesText.name(),
+    description: undefined,
+    renderPickList: true,
```

Comment:
pick list that pulls data from GeologicTimePeriod. We can already do that as a user > schema config > choose field i.e text1 in CO table > create pick list > type entries table > size limit (-1) in the code or 500 in UI > save
Need to do this in the system. 

If we query on the example text1 CO we have a pick list